sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -956,7 +979,18 @@ object Radar {
                             return
                         }
 
-                        verificationManager.getIntegrityToken(config.googlePlayProjectNumber, config.nonce) { integrityToken, integrityException ->
+                        if (config.nonce == null) {
+                            handler.post {
+                                callback?.onComplete(RadarStatus.ERROR_UNKNOWN)
+                            }
```

Comment:
Let's add an error log for debugging 

`logger.e("Error missing nonce for trackVerified", RadarLogType.SDK_ERROR)`